README.md

```diff
@@ -260,6 +260,78 @@ Now, any script you defined in the ScriptsFunction will be added to the HTML tog
 
 > Tip: You could use it together with useShouldHydrate to disable Remix scripts in certain routes but still load scripts for analytics or small features that need JS but don't need the full app JS to be enabled.
 
+
+### StructuredData
+
+If you need to include structured data (JSON-LD) scripts on certain routes, you can use the `StructuredData` component together with the `HandleStructuredData` type or `StructuredDataFunction` type.
+
+In the route you want to include the structured data, add a `handle` export with a `structuredData` method, this method should implement the `StructuredDataFunction` type.
+
+```ts
+import type { WithContext, BlogPosting } from 'schema-dts';
+
+// export the handle with the correct type:
+export let handle: HandleStructuredData<LoaderData> = {
+  structuredData: (data: LoaderData) => {
+    try {
+      let { post } = data;
+      
+      let postSchema: WithContext<BlogPosting> = {
+        '@context': 'https://schema.org',
+        '@type': 'BlogPosting',
+        datePublished: post.published,
+        mainEntityOfPage: {
+          '@type': 'WebPage',
+          '@id': post.postUrl,
+        },
+        image: post.featuredImage,
+        author: {
+          '@type': 'Person',
+          name: post.authorName,
+        },
+      };
+
+      return postSchema;
+    } catch (e: unknown) {
+      console.error(e);
+      return [];
+    }
+  },
+};
+```
+
+Then, in the root route, add the `StructuredData` component together with the Remix's Scripts component, usually inside a Document component.
+
+```tsx
+import { Links, LiveReload, Meta, Scripts, ScrollRestoration } from "remix";
+import { StructuredData } from "remix-utils";
+
+type Props = { children: React.ReactNode; title?: string };
+
+export function Document({ children, title }: Props) {
+  return (
+    <html lang="en">
+      <head>
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        {title ? <title>{title}</title> : null}
+        <Meta />
+        <Links />
+      </head>
+      <body>
+        {children}
+        <ScrollRestoration />
+        <Scripts />
+        <StructuredData />
```

Comment:
Is it ok to render this at the end of the `<body>`? Shouldn't it be inside `<head>`?